doc/CHANGES.md

```diff
@@ -4,6 +4,12 @@
 
 ## Upgrade Notes
 
+* The `InfoRequest` method `law_used_human` has been deprecated and will be
+  removed in a future release. The new method `legislation` has been supplied
+  instead which is an Legislation instance with an `#to_s` method equivalent to
```

Comment:
Minor:

* s/an Legislation instance/a Legislation instance`
* s/an `#to_s` method/a `#to_s` method/

Reply:
Changed in 40c0bfcc